src/lib/components/submit/SubmitReview.svelte

```diff
@@ -30,7 +33,14 @@
 	const dispatch = createEventDispatcher();
 
 	const onSubmit = async () => {
-		const { result, proposalId } = await submitMotionProposal({
+		let submitProposal;
```

Comment:
nth: it can be a `const`.

`const submitProposal = $store?.metadata?.proposalAction === 'AddOrRemoveNodeProvider' ? submitAddNodeProviderProposal : submitMotionProposal;`